src/dapps/provider-type.ts

```diff
@@ -0,0 +1,25 @@
+import { generateValidator, JSONSchema, ValidateFunction } from '../validation'
+
+/**
+ * Different supported providers
+ * @alpha
+ */
+export enum ProviderType {
+  INJECTED = 'injected',
+  FORTMATIC = 'formatic',
+  NETWORK = 'network',
+  WALLET_CONNECT = 'wallet_connect'
+}
+
+/**
+ * @alpha
+ */
+export namespace ProviderType {
+  export const schema: JSONSchema<ProviderType> = {
+    type: 'string',
+    enum: Object.values(ProviderType)
```

Comment:
Does this work? I've never added a Iterable to an enum, only arrays

Reply:
you have one here: https://github.com/decentraland/common-schemas/blob/main/src/dapps/network.ts

Reply:
https://github.com/decentraland/common-schemas/blob/main/src/dapps/network.ts

Reply:
we can change it to:
```ts
  export const schema: JSONSchema<ProviderType> = {
    type: 'string',
    enum: [
      ProviderType.INJECTED,
      ProviderType.FORTMATIC,
      ProviderType.NETWORK,
      ProviderType.WALLET_CONNECT,
    ]
  ]
```